From: Michael S. Tsirkin <mst@redhat.com>
Subject: [PATCH] vhost: copy_to_user -> __copy_to_user
References: FATE#311977
Git-commit: dfe5ac5b18be5b10d01a17e734a9905c0def6088
Patch-mainline: v2.6.37

We do access_ok checks at setup time, so we don't need to
redo them on each access.

Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
Acked-by: Bruce Rogers <brogers@suse.com>
---
 drivers/vhost/vhost.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: b/drivers/vhost/vhost.c
===================================================================
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -1258,7 +1258,7 @@ static int __vhost_add_used_n(struct vho
 
 	start = vq->last_used_idx % vq->num;
 	used = vq->used->ring + start;
-	if (copy_to_user(used, heads, count * sizeof *used)) {
+	if (__copy_to_user(used, heads, count * sizeof *used)) {
 		vq_err(vq, "Failed to write used");
 		return -EFAULT;
 	}
